core/templates/pages/donate-page/thanks-for-subscribing-modal.component.spec.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for ThanksForSubscribingModalComponent.
```

Comment:
```suggestion
 * @fileoverview Unit tests for thanks for subscribing modal component.
```

Reply:
Looking at the codebase, it seems we follow the pattern of using the actual component name. I've updated per suggestion.

Reply:
Ditto elsewhere.